website/content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,236 @@
+  | envsubst \
+  | eksctl create cluster -f -
+```
+
+This guide uses a regular (un-managed) node group to host Karpenter.
+
+Karpenter itself can run anywhere, including on self-managed node groups, [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), or [AWS Fargate](https://aws.amazon.com/fargate/).
+
+Karpenter will provision new traditional instances on EC2. 
+
+Additionally, the configuration file sets up an [OIDC
+provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens),
+necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
+of communicating with identity providers. 
+
+### Setup Authentication from Kubernetes to AWS (IRSA)
+
+IAM Roles for Service Accounts (IRSA) maps Kubernetes resources to roles
+(permission sets) on AWS. 
+
+First, define a role using the template below. It provides full access to EC2,
+and limited access to other services such as EKS and Elastic Container Registry
+(ECR).
+
+```bash
+# Creates IAM resources used by Karpenter
+TEMPOUT=$(mktemp)
+curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/docs/aws/karpenter.cloudformation.yaml > $TEMPOUT \
+&& aws cloudformation deploy \
+  --stack-name Karpenter-${CLUSTER_NAME} \
+  --template-file ${TEMPOUT} \
+  --capabilities CAPABILITY_NAMED_IAM \
+  --parameter-overrides ClusterName=${CLUSTER_NAME}
+```
+
+Second, create the mapping between Kubernetes resources and the new IAM role. 
+
+```bash
+# Add the Karpenter node role to your aws-auth configmap, allowing nodes with this role to connect to the cluster.
+eksctl create iamidentitymapping \
+  --username system:node:{{EC2PrivateDNSName}} \
+  --cluster  ${CLUSTER_NAME} \
+  --arn arn:aws:iam::${AWS_ACCOUNT_ID}:role/KarpenterNodeRole-${CLUSTER_NAME} \
+  --group system:bootstrappers \
+  --group system:nodes
+```
+
+Now, Karpenter can send requests for new EC2 instances to AWS. 
+
+### Install Karpenter Helm Chart
+
+Use helm to deploy Karpenter to the cluster. 
+
+We created a Kubernetes service account when we created the cluster using
+eksctl. Thus, we don't need the helm chart to do that.
+
+```bash
+helm repo add karpenter https://awslabs.github.io/karpenter/charts
+helm repo update
+helm upgrade --install karpenter karpenter/karpenter \
+  --namespace karpenter --set serviceAccount.create=false
+```
+
+### Provisioner
+
+A single Karpenter provisioner is capable of handling many different pod
+shapes. In other words, Karpenter eliminates the need to manage many different
+node groups. Karpenter makes scheduling and provisioning decisions based on pod
+attributes such as labels and affinity. 
+
+Create a simple default provisioner using the command below. This provisioner
+provides instances with the default certificate bundle, and the control plane
+endpoint url. 
+
+Importantly, the `ttlSecondsAfterEmpty` value configures Karpenter to
+deprovision empty nodes. 
```

Comment:
It may be worth to include `TTLSecondsUntilExpired` here as well (which recycles nodes automatically after some TTL). 

Additionally we could include the fact if someone doesn't want these two features, they can simply leave it out of the provisioner spec.

Reply:
added some desc of expiration and a link to the prov-crd file